CHANGELOG.md

```diff
@@ -63,6 +63,7 @@
 
 #### OCI / Docker Compatibility
 
+- A new command `get-login-password` that allows the user to retrieve their cli token.
```

Comment:
Please move this to a new `Changes since last release` section, above the 3.11.0 changelog.

Where you have it at the moment is the changelog for 3.11.0, which has already been released.

Reply:
Addressed in new commit.